flink-runtime/src/main/java/org/apache/flink/runtime/state/heap/HeapValueState.java

```diff
@@ -31,7 +31,7 @@
  * @param <N> The type of the namespace.
  * @param <V> The type of the value.
  */
-class HeapValueState<K, N, V> extends AbstractHeapState<K, N, V>
+public class HeapValueState<K, N, V> extends AbstractHeapState<K, N, V>
```

Comment:
Please revert the visibility if you adjust tests as I suggested.

ditto: other state classes